stage1/init/init.go

```diff
@@ -248,6 +250,11 @@ func stage1() int {
 
 func main() {
 	flag.Parse()
+
+	if !debug {
```

Comment:
that's fine

On Mon, Apr 6, 2015 at 12:02 PM, Eugene Yakubovich <notifications@github.com
> wrote:

> In stage1/init/init.go
> <https://github.com/coreos/rkt/pull/731#discussion_r27820405>:
>
> > @@ -248,6 +250,11 @@ func stage1() int {
> >
> >  func main() {
> >  	flag.Parse()
> > +
> > +	if !debug {
>
> do you mind if I update stage0 to be like this? why do the double flip?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/731/files#r27820405>.
>